projects/igniteui-angular/src/lib/grid/column.component.ts

```diff
@@ -409,8 +850,15 @@ export class IgxColumnComponent implements AfterContentInit {
         this.updateHighlights(oldIndex, newIndex);
         return true;
     }
-
-    public unpin(index?) {
+/**
+ * Unpins the column.
```

Comment:
Unpins the column and place it at the provided index in the unpinned area. Defaults to index 0 if not provided.